### ISSUE TITLE
NSLocalizedString throwing exception.

### ISSUE THREAD

**Issue by JonSmart** [arnesson/cordova-plugin-firebase#814]
`NSString *description = NSLocalizedString([command argumentAtIndex:1 withDefault:@"No Message Provided"], nil);`

In iOS is throwing an exception when calling `logEvent`?

Latest build version 1.1.1?

**Comment by JonSmart** [comment]
I am just working through testing a fix with:

```

        @try {
          NSString *description = NSLocalizedString([command argumentAtIndex:1 withDefault:@"No Message Provided"], nil);
          parameters = @{ NSLocalizedDescriptionKey: description };
        }
        @catch (NSException *execption) {
          parameters = [command argumentAtIndex:1];
        }
```

If it works I will create a PR.

**Comment by soumak77** [comment]
@JonSmart publishing of v1.1.1 failed.  Trying again.   I'll comment once published successfully.

**Comment by JonSmart** [comment]
@soumak77 no worries, would be good to get this resolved anyway?  As it currently crashes on iOS.

**Comment by soumak77** [comment]
@JonSmart `v1.1.1` has been published to npm

**Comment by JonSmart** [comment]
@soumak77 Just used the published 1.1.1 and this bug is still present.  Shall I do a PR for a fix?

**Comment by soumak77** [comment]
fixed via #815
